skll/data/readers.py

```diff
@@ -108,29 +108,29 @@ class Reader(object):
         Whether or not a FeatureHasher should be used to
         vectorize the features.
 
-    num_features : int, default=None
+    num_features : Optional[int], default=None
         If using a FeatureHasher, how many features should the
         resulting matrix have?  You should set this to a power
         of 2 greater than the actual number of features to
         avoid collisions.
 
-    logger : logging.Logger, default=None
+    logger : Optional[logging.Logger], default=None
         A logger instance to use to log messages instead of creating
         a new one by default.
     """
 
     def __init__(
         self,
         path_or_list: Union[PathOrStr, FeatureDictList],
-        quiet=True,
-        ids_to_floats=False,
-        label_col="y",
-        id_col="id",
+        quiet: bool = True,
+        ids_to_floats: bool = False,
+        label_col: Optional[str] = "y",
```

Comment:
Optional not needed unless it can become None later, since it has a default value

Reply:
Yep, that's why I changed it. We assign `None` to it later.